### ISSUE TITLE
TypeError: option.isSelected is not a function

### ISSUE THREAD

**Issue by tejasmanohar** [maxchehab/phelia#20]
```
          <MultiSelectMenu action="selection" placeholder="A placeholder">
            <OptionGroup label="an option group">
              <Option value="option-a" selected>
                option a
              </Option>
              <Option value="option-b" selected>
                option b
              </Option>
              {/* <Option value="option-b">option b</Option>
              <Option value="option-c">option c</Option> */}
            </OptionGroup>
          </MultiSelectMenu>
```

```
TypeError: option.isSelected is not a function
```

Context: App Home

**Comment by maxchehab** [comment]
Fixed in `0.1.6`

**Comment by tejasmanohar** [comment]
Thanks :D
